master/usage/configuration/mtu.md

```diff
@@ -83,6 +83,9 @@ Example CNI configuration
 }
 ```
 
+By default, CNI MTU and `FELIX_IPINIPMTU` derive their value from `veth_mtu` ConfigMap
```

Comment:
I think we need a bit that says "If using a Kubernetes self-hosted manifest to install Calico, ... "

This section isn't kubernetes specific, so by adding Kubernetes specific data we need to make that clear. To be honest, we probably don't _need_ this section at all - I think the comments in the manifests themselves should make this clear. wdyt?